<?xml version="1.0"?>
<launch>
  <node pkg="ndt_fuser" type="ndt_fuser" name="laser_fuser_2d" output="screen">
	<param name="laser_topic" value="/laserscan" />
	<param name="matchLaser" value="true" />
	
	<param name="size_x_meters" value="60" />
	<param name="size_y_meters" value="60" />
	<param name="size_z_meters" value="0.1" />
	<param name="resolution" value="0.2" />
	
	<param name="sensor_range" value="20." />
	
	<!-- Specific sensor offset parameters for the example file -->
	<param name="sensor_pose_x" value="0.695" />
	<param name="sensor_pose_y" value="-0.01" />
	<param name="sensor_pose_t" value="-0.0069813" />

	<param name="map_directory" value="$(find ndt_fuser)/maps/" />	
	<param name="map_name_prefix" value="basement2d" />

	<param name="beHMT" value="false" />
	<param name="useOdometry" value="true" />
	<param name="odometry_topic" value="/vmc_navserver/odom" />
	<param name="visualize" value="true" />
    </node>
</launch>
